packages/cdktf-cli/lib/get/generator/models/attribute-model.ts

```diff
@@ -75,6 +75,8 @@ export class AttributeModel {
     if (this._name === "equals") return "equalTo";
     // `node` is already used by the Constructs base class
     if (this._name === "node") return "nodeAttribute";
+    // FIXME: this is only temporary and will not work when this attribute is actually used.
+    if (this._name === "provider") return "providerAttribute";
```

Comment:
Seems unrelated. Did this come up?

Reply:
The latest commit is a "wip" commit 😁 So yeah, that'll go again, but currently I need this for the provider I'm testing with (some private internal one which is also currently wip, so that's why it had this attribute I guess).